packages/configs/eslint/configs/jest.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  plugins: ['@typescript-eslint', 'jest'],
+  plugins: ['@typescript-eslint', 'import', 'jest'],
   extends: ['plugin:jest/recommended'],
   rules: {
     // turn off for test file linting
     '@typescript-eslint/no-unsafe-assignment': 'off',
-
+    // ignore dev dependencies used in test files
+    'import/no-extraneous-dependencies': 'off',
```

Comment:
Let's configure the base rule to use a glob pattern ignore devDeps on matches against test files or use the `packageDir` option to specify the monorepo root _package.json_ as well as the current directory _package.json_, e.g.:
```
'import/no-extraneous-dependencies': ['error', { packageDir: ['.', '../..'] },
```

Reply:
Updated. `yarn lint` works as expected but I'm seeing `The package.json file could not be found.` with VScode, not sure if that's just a problem with my machine though